setup.py

```diff
@@ -79,8 +82,8 @@ def run(self):
       author_email='project-ganga-developers@cern.ch',
       license='GPL v2',
       scripts=['bin/ganga'],
-      package_dir={'': 'python'},
-      packages=find_packages('python'),
+      package_dir={'ganga': 'ganga'},
+      packages=allPackages,
```

Comment:
I believe that these two lines can now be simply:
```python
packages=find_packages()
```

Reply:
yes